UniversityTimetablingSystem/backend/university_timetabling/timetable/views.py

```diff
@@ -0,0 +1,160 @@
+# backend/university_timetabling/timetable/views.py
+
+from django.shortcuts import render
+from django.http import JsonResponse, HttpResponseBadRequest, HttpResponseServerError
+from django.views.decorators.csrf import csrf_exempt
+import os
+import json
+import traceback
+import subprocess
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+project_root = os.path.abspath(os.path.join(current_dir, '../../../bin'))
+
+@csrf_exempt
+def add_instructor(request):
+    if request.method == 'POST':
```

Comment:
use the opposite condition for returning Invalid request method. So your branching will decrease